core/src/test/scala/anorm/H2Database.scala

```diff
@@ -5,7 +5,7 @@ import scala.util.Random
 
 trait H2Database {
   def withH2Database[R](block: Connection => R): R = {
-    val url = "jdbc:h2:mem:test" + Random.alphanumeric.take(6).mkString("")
+    val url = "jdbc:h2:mem:test" + Random.alphanumeric.take(6).mkString("") + ";DATABASE_TO_UPPER=false"
```

Comment:
After some research into acolyte, it doesn't seem like one can specify a different qualified name and alias using `withLabel`. We need to have a setup with 2 columns with the same qualified name, but *different aliases* in order to get a failing test. Otherwise the test will succeed even without the suggested change and doesn't prove anything.

When the alias is not found, it will fall back to using the qualified name, which will work fine *provided* there is a *single* column with this qualified name.

Please advise how to proceed.